front/public/locales/en/errors.json

```
@@ -217,7 +217,8 @@
       "InvalidUrl": "Invalid url '{{url}}'"
     },
     "work_schedule": {
-      "NameAlreadyUsed": "A group of work schedules with '{{name}}' already exists"
+      "NameAlreadyUsed": "A group of work schedules with '{{name}}' already exists",
+      "WorkScheduleGroupNotFound": "No such work schedule group with id={{id}}"
```

To keep it harmonized with other trads (same for fr)

```suggestion
      "WorkScheduleGroupNotFound": "No such work schedule group with id '{{id}}'"
```